packages/ses/src/module-instance.js

```diff
@@ -326,16 +316,10 @@ export const makeModuleInstance = (
     activate();
   }
 
-  let optFunctor = performEval(
-    functorSource,
+  let optFunctor = compartment.evaluate(functorSource, {
```

Comment:
https://github.com/Agoric/SES-shim/blob/1c8e7066c195d212c06a31cd91efb47c2b6a3e76/packages/transform-module/src/main.js#L157-L163
defines the static record produced by our transform. It is static and it is a record and it does represent a module. However, the name `moduleStaticRecord` is too close to `StaticModuleRecord`. The code in this PR (especially here) uses such a `moduleStaticRecord` to build a `staticModuleRecord`. However, the code here consistently refers to it as a `moduleAnalysis`. Could we adopt that naming convention in the transform-module package as well?

Reply:
Yes, I would like to thread the names we landed on in SES back up to transform-module. I also identified a vestige in the module functor calling convention that we can remove.